__tests__/app.test.js

```diff
@@ -36,3 +36,37 @@ describe("GET /api/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: respond with JSON object of all /api/articles/:article_id endpoints", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const {articles} = body;
+
+        articles.forEach((articles) => {
+          expect(articles).toHaveProperty("article_id", expect.any(Number));
+          expect(articles).toHaveProperty("title", expect.any(String));
+          expect(articles).toHaveProperty("topic", expect.any(String));
+          expect(articles).toHaveProperty("author", expect.any(String));
+          expect(articles).toHaveProperty("body", expect.any(String));
+          expect(articles).toHaveProperty("created_at", expect.any(String));
+          expect(articles).toHaveProperty("votes", expect.any(Number));
+          expect(articles).toHaveProperty("article_img_url", expect.any(String));
```

Comment:
You test with the actual data of the article rather than just the shape of the object since you know it is article 1